README.md

```diff
@@ -39,10 +39,17 @@ These instructions will get you a copy of the Quarkus.io website up and running
 
 To write a blog:
 
-- create an author ent ry in [_data/authors.yaml](https://github.com/quarkusio/quarkusio.github.io/blob/develop/_data/authors.yaml)
+- create an author entry in [_data/authors.yaml](https://github.com/quarkusio/quarkusio.github.io/blob/develop/_data/authors.yaml)
     - `emailhash` is used to fetch your picture form the Gravatar service
 - create an blog entry under [_posts](https://github.com/quarkusio/quarkusio.github.io/tree/develop/_posts)
     -the file name is `yyyy-mm-dd-slug.adoc`
+- `tags` should be used with some care as an archive page is created for of them. Below are some basic rules to try follow:
+  - `quarkus-release` used for Quarkus release blogs
```

Comment:
I would prefer the simpler "release" I think.

Reply:
just done to in future maybe have platform-release, etc. but fine just using `release` for main quarkus and if we start having others we just add them.